demo-extended/src/main/java/com/novoda/simplechromecustomtabs/demo/linkify/UrlSpanFactory.java

```diff
@@ -0,0 +1,25 @@
+package com.novoda.simplechromecustomtabs.demo.linkify;
+
+import android.net.Uri;
+import android.text.style.URLSpan;
+import android.view.View;
+
+class UrlSpanFactory {
+
+    private final OnWebLinkClickedListener onWebLinkClickedListener;
+
+    UrlSpanFactory(OnWebLinkClickedListener onWebLinkClickedListener) {
+        this.onWebLinkClickedListener = onWebLinkClickedListener;
+    }
+
+    URLSpan createSpan(final String url) {
```

Comment:
just to be consistent with the naming that seems to be throughout the pr: `createSpanFor/With`?